### PR TITLE
Added support to search for a senator

### PR DESCRIPTION
**Pull request by Taz17**
This aims to solve [need easy way to search for senators #1198](https://github.com/openaustralia/publicwhip/issues/1198) issue.

I made changes to the UI to give the user an example of how to search for a senator. It now rotates between the different examples rather than adding `or Senator for Tasmania` since the width of the randomly chosen placeholder is unpredictable

UI_changes

The search supports all ways of representing a constituency e.g. `Queensland` or `QLD`

However it does not address this comment https://github.com/openaustralia/publicwhip/issues/1198#issuecomment-682216684 where I believe the user is trying to search for the senator by postcode? Maybe `Senator for 3166`

### CODE REVIEW

**Comment by Taz17** [comment]
Result: 

Accepts both `Senate for ...` and `Senator for ....` of course case-insensitive 

image

**Comment by Taz17** [comment]
Not sure why the build is failing, it passes both `rake tasks` and `rubocop` checks

**Comment by mlandauer** [comment]
Well I can see from https://app.travis-ci.com/github/openaustralia/publicwhip/builds/246194478 that the CI is failing because the regression tests are failing. The most likely explanation is that what's running locally is not the same as what's running on travis! Maybe you forgot to commit something?

**Comment by Taz17** [comment]
This is a bit weird because I don't have any uncommitted changes. I even went as far as pulling the latest changes from `openaustralia:master` and rerun the `rake` and `rubocop` to have them both pass.

Here is what I get when I run `bundle exec rake`, `bundle exec rubocop`, and `git status`

```
mutaz@mutaz:~/Documents/repos/publicwhip$ bundle exec rake
/home/mutaz/.rbenv/versions/3.0.0/bin/ruby -I/home/mutaz/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/lib:/home/mutaz/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/rspec-support-3.11.0/lib /home/mutaz/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/rspec-core-3.11.0/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb

Randomized with seed 28701
 386/386 |============================================================== 100 ==============================================================>| Time: 00:00:09 

Finished in 9.73 seconds (files took 2.78 seconds to load)
386 examples, 0 failures

Randomized with seed 28701
Coverage report generated for RSpec to /home/mutaz/Documents/repos/publicwhip/coverage. 2925 / 3260 LOC (89.72%) covered.
mutaz@mutaz:~/Documents/repos/publicwhip$ bundle exec rubocop
Inspecting 133 files
.....................................................................................................................................

133 files inspected, no offenses detected
mutaz@mutaz:~/Documents/repos/publicwhip$ git status
On branch senator_search
Your branch is up to date with 'origin/senator_search'.

nothing to commit, working tree clean
mutaz@mutaz:~/Documents/repos/publicwhip$
```

[Screenshot of the terminal session](https://user-images.githubusercontent.com/86769131/154872802-fab891d5-3b0f-468a-839e-6babebab0fee.png)

File: spec/fixtures/static_pages/.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
 <meta charset='utf-8'>
 <meta content='IE=edge' http-equiv='X-UA-Compatible'>
 <meta content='width=device-width, initial-scale=1' name='viewport'>
+<meta content='1234' property='fb:admins'>
```
Comment (mlandauer) [inline spec/fixtures/static_pages/.html:7]: @Taz17 this looks very suspicious. I would guess this is the source of the problem. Why was this committed?

Comment (Taz17) [reply spec/fixtures/static_pages/.html:7]: Those were changes made automatically after `bundle exec rake` was run. I manually anything under `spec/*`

Comment (mlandauer) [reply spec/fixtures/static_pages/.html:7]: It those get changed, that means the regression tests failed!

Comment (Taz17) [reply spec/fixtures/static_pages/.html:7]: Yes definitely, that was in commit [42ba8f1](https://github.com/openaustralia/publicwhip/pull/1329/commits/42ba8f1128bc2e5690e2ccfe6ab3f0e77a1d9993)

I then re-ran the rake and it added more changes. Here [33cca66](https://github.com/openaustralia/publicwhip/pull/1329/commits/33cca66676ffbbda81fe3e73e11b5f7561870532)

But its passing everything now?

Comment (mlandauer) [reply spec/fixtures/static_pages/.html:7]: Why is something to do with a meta tag to do with facebook getting updated in a PR to do with searching?

Comment (Taz17) [reply spec/fixtures/static_pages/.html:7]: I don't know, all the changes that I made are regarding search.

What could be the cause of the meta tag getting updated?

As a side note. I just ran the rake tasks on the branch ([policy_filter_search_js](https://github.com/Taz17/publicwhip/tree/policy_filter_search_js)) to see why PR https://github.com/openaustralia/publicwhip/pull/1332 was failing the rake tasks and I got the same changes for the facebook meta tags... very odd

Comment (Taz17) [reply spec/fixtures/static_pages/.html:7]: I also get the same meta tag changes if i run `bundle exec rake` on the master branch (note: i never code on the master branch)

Comment (mlandauer) [reply spec/fixtures/static_pages/.html:7]: What you need to do is understand where that meta tag is being generated and then figure out what you're doing to cause that to be different on **your** box vs someone else's box.

**Pull request closed without merging**